modules/system/classes/UpdateManager.php

```diff
@@ -234,15 +233,15 @@ public function requestUpdateList($force = false)
         }
 
         $params = [
-            'core' => $this->getHash(),
+            'core'    => $this->getHash(),
             'plugins' => serialize($versions),
-            'themes' => serialize($themes),
-            'build' => $build,
-            'force' => $force
+            'themes'  => serialize($themes),
+            'build'   => $build,
+            'force'   => $force
         ];
 
         $result = $this->requestServerData('core/update', $params);
-        $updateCount = (int) array_get($result, 'update', 0);
+        $updateCount = (int)array_get($result, 'update', 0);
```

Comment:
Space between typecast and value being cast please

Reply:
Phpstorm that ident

Reply:
Revert it please.